### DISCUSSION TITLE
Organización apartados de discusión

### DISCUSSION THREAD

**Discussion by edreivaj** [PEA-IA-DS-Proyecto-2/Project-2---Predict_Ratings_based_on_text_statements#4] [General]
He creado una estructura en este espacio de discusión donde poder aportar detalles de los  avances  que hacemos en cada punto que nos corresponda liderar cuando lo decidamos antes  de fin de año.

Dicha estructura está compuesta por una categoría relacionada con cada uno de los 6 puntos que se indican en el proyecto.

Creo que puede ser útil para estructurar la información. En consecuencia, aspectos generales del proyecto lo escribiríamos en el apartado General y en el resto de apartados escribiríamos lo que corresponda a ese apartado concreto.

Seguimos!!!!

**Comment by SqueezeU** [comment]
ok - ya me explicas en el Team meeting
